lib/roman-numerals.js

```diff
@@ -0,0 +1,15 @@
+function romanNumeralConverter(num) {
```

Comment:
this is really hard to read. i'm not sure what each variable represents in the context of your solution by reading through your code. could you provide an overview of what's going on?